### PR TITLE
homework

### PR DESCRIPTION
**Pull request by pilipenok1995**

### CODE REVIEW

File: t2.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            print('{0}   *'.format(ab[i]*ab[i+1]))
+   
+    print("---------")
+print(ab)
+# 4 task
+sum = 0
+for e in ab:
+    if e % 2 != 0:
+        sum += e
+print('\n4.task =   {0} '.format (sum))
```
Comment (MikitaTsiarentsyeu) [inline t2.py:38]: could've use a single cycle

File: t2.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+text = 'five thiteen two eleven seventeen two one thirteen ten four eigtht five nineteen'
+a = text.split()
+d = {
```
Comment (MikitaTsiarentsyeu) [inline t2.py:3]: and why do we need this dictionary?

File: t2.py (hunk 1 above)
Comment (pilipenok1995) [reply t2.py:38]: it is more easy

File: t2.py (hunk 2 above)
Comment (pilipenok1995) [reply t2.py:3]: bcause i need to get int.number for string

File: t2.py (hunk 1 above)
Comment (pilipenok1995) [reply t2.py:38]: why?

**Pull request merged**